src/Microsoft.Diagnostics.EventFlow.Core/Implementations/HealthReporters/CsvHealthReporter.cs

```diff
@@ -290,9 +372,18 @@ private void Initialize(CsvHealthReporterConfiguration configuration, INewReport
             // Prepare the configuration, set default values, handle invalid values.
             this.Configuration = configuration;
 
-            // Create a default throttle
+            // Create a default throttle.
             this.throttle = new TimeSpanThrottle(TimeSpan.FromMilliseconds(DefaultThrottlingPeriodMsec));
 
+            // Set the file size for csv health report. Minimum is 1MB. Default is 8192MB.
+            this.SingleLogFileMaximumSizeInBytes = (long)(configuration.SingleLogFileMaximumSizeInMBytes > 0 ? configuration.SingleLogFileMaximumSizeInMBytes : 8192) * 1024 * 1024;
+
+            // Set default value for retention days for the logs. Minimum value is 1.
```

Comment:
Minimum value is 30

Reply:
@karolz-ms , do you really mean to make the minimum value to 30? Which sounds a bit aggressive. I think a minimum of 1 day but default to 30 days is more flexible.

Reply:
@karolz-ms , do you really mean to make the minimum value to 30? Which sounds a bit aggressive. I think a minimum of 1 day but default to 30 days is more flexible.